shared/message/planning/KickTo.proto

```diff
@@ -0,0 +1,28 @@
+/*
+ * This file is part of the NUbots Codebase.
+ *
+ * The NUbots Codebase is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The NUbots Codebase is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the NUbots Codebase.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright 2022 NUbots <nubots@nubots.net>
+ */
+syntax = "proto3";
+
+package message.planning;
+
+import "Vector.proto";
+
+message KickTo {
```

Comment:
I know it's an extra transformation but I feel like it might be better to have this target in world space rather than torso space. The place you're kicking too will generally be a place in the world (won't rotate as the torso rotates and moves)

Reply:
I don't think I can trust world space enough quite yet to do that

Reply:
I still don't want world space here yet, none of our measurements are in world space at this point (ie goals). And this functionality is still in development. The aim should be to mostly move what we currently have in the codebase over. We don't really want to get to the end and have to rewrite it because we've gone overboard with changes.
Once we can align with vectors from the torso, then we can look at moving it to some other space.